### ISSUE TITLE
Can’t see Android app on the Play Store 

### ISSUE THREAD

**Issue by Thovi98** [streamaserver/streama#949]
### Issue description
The Android app doesn’t seem to be on the Play Store anymore. I don’t know if it is normal ?

And it could be very nice if it was pulled on Fdroid too.

**Comment by coamtaker618** [comment]
I just noticed this as well @Thovi98    This type of service works great with an Android app!   

However the android project seems to have not been touched in a while >_>
https://github.com/streamaserver/streama-app

Last release was Sept 2018 :(    But if there's a blog or some official statement, it would be nice to hear what's going on :(

**Comment by dularion** [comment]
Unfortunately they have rejected it again: 

![image](https://user-images.githubusercontent.com/936076/110169784-7baa0200-7df9-11eb-86f9-2570d615a7a3.png)

This might or might not have to do with the actual screenshots that are uesd. I try using generic images instead and see if that helps

**Comment by dularion** [comment]
Yeah i could put it up on f-droid i guess. it requires some kind of apk-self-hosted API which i could play around with.

**Comment by brianhenson** [comment]
you could use things like big buck bunny and similar to bypass the copyright issue
